.github/workflows/actions/deploy/action.yml

```diff
@@ -49,40 +49,40 @@ runs:
        - name: Get Short SHA
          id: sha
          shell: bash
-         run: echo ::set-output name=short::$(echo "${{ inputs.sha }}" | cut -c -7)
+         run: echo "short=$(echo "${{ inputs.sha }}" | cut -c -7)" >> $GITHUB_OUTPUT
 
        - name: Setup Environment Variables
          id:  variables
          shell: bash
          run: |
              if [ "${{inputs.environment }}" == "Development" ]
              then
-                 echo ::set-output name=control::$(echo "dev" )
-                 echo ::set-output name=healthcheck::$(echo "${{env.PAAS_APPLICATION_NAME}}-dev" )
-                 echo ::set-output name=key::"api.dev.terraform"
-                 echo ::set-output name=docker_image::"${{env.DOCKER_REPOSITORY}}:sha-${{ steps.sha.outputs.short}}"
+                 echo "control=$(echo "dev" )" >> $GITHUB_OUTPUT
```

Comment:
We don't actually need `echo` here and in subsequent lines. Can we simplify?

Reply:
Tried this but it failed. This is using the standard format in the documentation.